packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -181,6 +182,19 @@ def get_ps_output():
         assert len(get_ps_output().split()) <= 4, 'UCR app has more than 4 processes running in its pid namespace'
 
 
+def test_memory_profiling(dcos_api_session):
+    # Test that we can fetch raw memory profiles
+    master_ip = dcos_api_session.masters[0]
+    profile_id = dcos_api_session.get(
+        '/memory-profiler/start?duration=1secs', host=master_ip, port=5050).json()['id']
+
+    time.sleep(2)
```

Comment:
Why do we `sleep`. I'd prefer if we wouldn't sleep at all. If you still want to sleep, please explain what drives this particular timeout so it can be understood should this ever become flaky.

Reply:
We sleep because we wait until jemalloc collects some data. I would however explicitly call `/stop` to avoid any flakiness.

Reply:
If you care about actual data being returned, maybe add some test that the response is not empty or some other metric of usefulness.

Reply:
Fixed.